src/string.rs

```diff
@@ -39,6 +39,7 @@ pub fn rewrite_string<'a>(s: &str, fmt: &StringFormat<'a>) -> Option<String> {
 
     let graphemes = UnicodeSegmentation::graphemes(&*stripped_str, false).collect::<Vec<&str>>();
     let indent = fmt.offset.to_string(fmt.config);
+    let punctuation = ",.:;";
```

Comment:
It costs us an extra type conversion when we do the contains(graphemes[cur_end - 1]) function, do you still want to go forward with this?